docs/dev-env-first-time-contributors.md

```diff
@@ -270,9 +270,12 @@ Now you are ready for [Step 2: Get Zulip Code.](#step-2-get-zulip-code)
    do this.
 2. Open Terminal (macOS/Ubuntu) or Git BASH (Windows; must
    **run as an Administrator**).
-3. In Terminal/Git BASH, clone your fork:
+3. In Terminal/Git BASH, clone your fork to your local machine using -c
+flag to set the git pull mode to be rebase by default in the repository
+configuration:
```

Comment:
This feels at the same time too long (it's kinda distracting) and too short (it doesn't really explain why we do this)

I think we want to avoid writing this explanation for what it does in like 5 places; I'd rather have a link to a subsection of `docs/git-guide.md` that has a longer, more complete explanation about why, and then have the individual references have something short.